src/commands/config/set.js

```diff
@@ -0,0 +1,88 @@
+const { TwilioCliError } = require('@twilio/cli-core/src/services/error');
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const { flags } = require('@oclif/command');
+
+const { availableConfigs } = require('./list');
+
+class ConfigSet extends BaseCommand {
+  async run() {
+    await super.run();
+    let isError = true;
+    let isUserConfigUpdated = false;
+    for (const flag of availableConfigs) {
+      if (this.flags[flag] !== undefined) {
+        isError = false;
+        this.preWarnings(flag);
+        if (this.flags[flag] === '') {
+          isUserConfigUpdated = await this.removeConfig(flag, isUserConfigUpdated);
+          continue;
+        }
+        if (await this.isOverwrite(flag)) {
+          this.userConfig[flag] = this.userConfig.sanitize(this.flags[flag]);
+          isUserConfigUpdated = true;
+        }
+      }
+    }
+    if (isError) {
+      throw new TwilioCliError(
+        `No configuration is added to set. Run "twilio configs:set --help" to see how to set a configurations.`,
+      );
+    }
+    if (isUserConfigUpdated) {
+      await this.saveConfiguration();
+    }
+  }
+
+  preWarnings(flag) {
+    if (flag === 'edge' && process.env.TWILIO_EDGE) {
+      this.logger.warn(`There is an environment variable already set for ${flag} : ${process.env.TWILIO_EDGE}`);
+    }
+  }
+
+  async removeConfig(flag, isUserConfigUpdated) {
+    if (this.userConfig[flag]) {
+      const verdict = await this.confirmDialog('Remove', flag);
+      if (verdict) {
+        isUserConfigUpdated = true;
+        this.userConfig[flag] = undefined;
+      }
+    } else {
+      this.logger.info(`There is no existing ${flag} to remove.`);
+    }
+    return isUserConfigUpdated;
+  }
+
+  async isOverwrite(flag) {
+    if (this.userConfig[flag]) {
+      return this.confirmDialog('Overwrite', flag);
+    }
+    return true;
+  }
+
+  async saveConfiguration() {
+    const configSavedMessage = await this.configFile.save(this.userConfig);
+    this.logger.info(configSavedMessage);
+  }
+
+  async confirmDialog(type, flag) {
+    const confirm = await this.inquirer.prompt([
+      {
+        type: 'confirm',
+        name: type,
+        message: `${type} existing ${flag} value?`,
+        default: false,
+      },
+    ]);
+    return confirm[type];
+  }
+}
+ConfigSet.description = 'sets a config to Twilio configuration.';
```

Comment:
Change to 'update Twilio CLI configuration'?

Reply:
updated.